masterlist.yaml

```diff
@@ -9774,3 +9774,54 @@ plugins:
         util: 'SSEEdit v4.0.1n RC4 for 4.0.2'
       - crc: 0xDA25CBFC
         util: 'SSEEdit v4.0.1n RC4 for 4.0.2'
+  - name: 'MissingEZs_AllExteriors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/23609'
+        name: 'Missing Encounter Zones Fixed on Nexus Mods'
+    group: *highPriorityGroup
+    after:
+      - 'RelightingSkyrim_SSE.esp'
+      - 'Populated Dungns Caves Ruins Legendary.esp'
+    tag:
+      - C.Encounter
+      - C.Location
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Realistic Water Two' ]
+        condition: 'active("RealisticWaterTwo.esp") and not active("MZEF_ExteriorsRW2Patch.esp")'
+  - name: 'MZEF_ExteriorsRW2Patch.esp'
```

Comment:
Missing Change from RWT
![image](https://user-images.githubusercontent.com/1944639/53518013-fa139100-3ac7-11e9-893c-88968b7e399d.png)

Reply:
how did that sneak past me? thanks! 😅

Reply:
Also 
![image](https://user-images.githubusercontent.com/1944639/53518934-fed94480-3ac9-11e9-8f7c-d5cf6eef6e58.png)

Reply:
SSEEdit Modgroups make it much easier to spot those.

Reply:
that's bizarre, I'm not seeing any of these conflicts with RW2 in SSEdit :S

Reply:
I'll see if I have an Older version.